edeploy-lxc

```diff
@@ -65,12 +65,30 @@ def get_dist(rootfs):
         rel = 'centos'
     if os.path.isfile(dr_path) and file(dr_path).read().find('Debian GNU/Linux 7') == 0:
         rel = 'debian'
+    if os.path.isfile(rr_path) and file(rr_path).read().find('Fedora release') == 0:
```

Comment:
The file to read is different on Debian than the one on CentOS / Redhat (dr_path and rr_path).

I'll upload a new version of the PR with your recommandation